src/logging.rs

```diff
@@ -125,13 +125,27 @@ impl log::Log for Logger {
     }
 
     fn log(&self, record: &log::Record<'_>) {
-        if self.enabled(record.metadata()) && record.target().starts_with("alacritty") {
-            let msg = format!(
-                "[{}] [{}] {}\n",
-                time::now().strftime("%F %R").unwrap(),
-                record.level(),
-                record.args()
-            );
+        if self.enabled(record.metadata()) &&
+           record.target().starts_with("alacritty")
+        {
+            let t = time::now();
+            let now = t.strftime("%F %R").unwrap();
+
+            let msg = if record.level() >= Level::Trace {
```

Comment:
Is this correct? I'd think that `record.level() >= Level::Trace` would be true for all levels.

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=e4b114623a1b14f715fbf552e60612b4

Reply:
I can never remember this...